metadrive/constants.py

```diff
@@ -270,6 +270,7 @@ class PGLineType:
     BROKEN = MetaDriveType.LINE_BROKEN_SINGLE_WHITE
     CONTINUOUS = MetaDriveType.LINE_SOLID_SINGLE_WHITE
     SIDE = MetaDriveType.BOUNDARY_LINE
+    BARRIER = MetaDriveType.BARRIER
```

Comment:
Please rename `BARRIER` to `GUARDRAIL` and `BARRIER THICKNESS` to `GUARDRAIL HEIGHT`